src/util/load.js

```diff
@@ -5,14 +5,10 @@ const Couch = require('../index');
 const debug = require('./debug')('util:load');
 const getConfiguredDbs = require('./getConfiguredDbs');
 
-var loaded = false;
-
 async function loadCouch() {
-  if (loaded) return;
   debug.trace('preload databases that have a configuration file');
   const configuredDbs = await getConfiguredDbs();
   configuredDbs.forEach((db) => Couch.get(db));
-  loaded = true;
 }
```

Comment:
is this safe?

Reply:
Yes, there is already a cache system implemented in `Couch.get`, so databases should not get instantiated twice. And `loadCouch` is only called once when the server starts.